examples/contacts.py

```diff
@@ -7,7 +7,8 @@
     raise EnvironmentError("RESEND_API_KEY is missing")
 
 # replace with some audience id
-audience_id: str = "ca4e37c5-a82a-4199-a3b8-bf912a6472aa"
+audience_id: str = "78b8d3bc-a55a-45a3-aee6-6ec0a5e13d7e"
+contact_id = "b5c95127-937b-4872-a765-06636e5f73da"
```

Comment:
should we add `:str` as the type hint? I'm in favor of keeping it consistent either we add it to this new line or remove `:str` from the line above

Reply:
good catch, fixed.